### DISCUSSION TITLE
Ocaml kernel for jupyter notebook

### DISCUSSION THREAD

**Discussion by Grasboulot** [microsoft/vscode-jupyter#12667] [Questions and Answers]
Hello, I can't use a jupyter notebook written in Ocaml in VS code. I am on wsl and I can't get the kernel to run my notebook. Do you have any idea how to do this?

Thanks in advance!

**Comment by DonJayamanne** [comment]
@Grasboulot please reply to this issue, https://github.com/microsoft/vscode-jupyter/issues/12669
